tests/sentry/notifications/utils/test_participants.py

```diff
@@ -331,10 +361,177 @@ def test_fallthrough(self):
+        )
+
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {
+                user_service.get_user(self.user3.id),
+                user_service.get_user(self.user.id),
+            },
+            ExternalProviders.SLACK: {
+                user_service.get_user(self.user3.id),
+                user_service.get_user(self.user.id),
+            },
+        }
+
+    @with_feature("organizations:streamline-targeting-context")
+    @with_feature("organizations:commit-context")
+    def test_send_to_suspect_committers_with_commit_context_feature_flag(self):
+        self.commit = self.create_sample_commit(self.user3)
+        event = self.store_event(
+            data={
+                "stacktrace": STACKTRACE,
+            },
+            project_id=self.project.id,
+        )
+
+        GroupOwner.objects.create(
+            group=event.group,
+            user=self.user3,
+            project=self.project,
+            organization=self.organization,
+            type=GroupOwnerType.SUSPECT_COMMIT.value,
+            context={"commitId": self.commit.id},
+        )
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {
+                user_service.get_user(self.user3.id),
+                user_service.get_user(self.user.id),
+            },
+            ExternalProviders.SLACK: {
+                user_service.get_user(self.user3.id),
+                user_service.get_user(self.user.id),
+            },
+        }
+
+    @with_feature("organizations:streamline-targeting-context")
+    @with_feature("organizations:commit-context")
+    def test_send_to_suspect_committers_no_owners_with_commit_context_feature_flag(self):
+        project_no_team = self.create_project(
+            name="No Team Project", organization=self.organization
+        )
+        self.commit = self.create_sample_commit(self.user3)
+        event = self.store_event(
+            data={
+                "stacktrace": {
+                    "frames": [
+                        {
+                            "function": "handledError",
+                            "abs_path": "Application.lol",
+                            "module": "io.sentry.example.Application",
+                            "in_app": True,
+                            "lineno": 39,
+                            "filename": "Application.lol",
+                        },
+                    ]
+                },
+            },
+            project_id=project_no_team.id,
+        )
+
+        GroupOwner.objects.create(
+            group=event.group,
+            user=self.user3,
+            project=self.project,
+            organization=self.organization,
+            type=GroupOwnerType.SUSPECT_COMMIT.value,
+            context={"commitId": self.commit.id},
+        )
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {
+                user_service.get_user(self.user3.id),
```

Comment:
`user3` is the suspect committer, no teams/users own the project

Reply:
`self.suspect_committer_user`